rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java

```diff
@@ -36,7 +35,7 @@
 	static final String CONTENT_TYPE = "application/json";
 
 
-	@Autowired
+	@Inject
```

Comment:
And why not use constructor injection here?  Or, asked another way, why are some classes do injection via the constructor and others via `@Inject`?

Reply:
Yeah I should circle back to this today. This was a punt because the `ValidationServiceImplTest` is opinionated in its initialization of ValidationServiceImpl.